### ISSUE TITLE
Give an option to not silence people when multiple persons are speaking

### ISSUE THREAD

**Issue by PureTryOut** [jitsi/jitsi-meet#8602]
### Description:

When multiple people are talking, currently all but 1 are sort of silenced and hard to understand. While I'm sure that can be useful, I'd like an option to disable this as for me it actually makes conversations hard to follow when people are cut out. It even silences people when someone that I personally have muted (well, volume set to as low as possible since the person is sitting next to me) is talking.

### Steps to reproduce:

1. Start a chat with 3 people
2. Have the other 2 people talk at the same time
3. Note that you can only hear 1 of them and the other is weirdly cut out

### Expected behavior:

Both people to be just as loud and not silenced

### Actual behavior:

One person is silenced/cut out, seemingly randomly decided which one.

### Server information:

- Jitsi Meet version: Not sure, I an instance I don't self host from https://meet.speakup.nl

### Client information:

- Browser / app version: Firefox
- Operating System: Alpine Linux

**Comment by damencho** [comment]
You can hear multiple people, there is no option to automate someone. Can you reproduce it on meet.jit.si? What about when using chromium, is it the same?

**Comment by damencho** [comment]
Please, when you have problems or questions use the community forum before opening new issues, thank you.

**Comment by saghul** [comment]
I think  what you see is AGC kicking in. This is what browsers do by default. You can try to load your conference with `#config.disableAGC` and see if you can feel the difference.

**Comment by PureTryOut** [comment]
AGC doesn't seem to be it sadly. I disabled it just now and the behavior hasn't changed. Besides, I don't have this issue in other video conferencing platforms like Google Meet. Basically it sounds like the other users are moving away from their mic or something. They don't get muted completely but the volume clearly gets a lot lower and the voices get unrecognizable.

I have not tried Chromium as for one I don't have it installed, but it's also really unstable on my system and has a lot of crashing tabs. Firefox is the only solid and reliable option for me.

A bit annoying that I would have to make an account on a forum while I already have a Github account to create issues and bug reports. I'm reporting what I think is a bug, I'm not asking for end-user support.

**Comment by damencho** [comment]
> A bit annoying that I would have to make an account on a forum while I already have a Github account to create issues and bug reports. I'm reporting what I think is a bug, I'm not asking for end-user support.

You can use your github account there.




> I'm reporting what I think is a bug

This is why we want people to report there so we can properly discuss it and decide whether it is a bug or no.
And someone from the community may have already seen that and help you, but those people are not monitoring the issue tracker to help you here.
